CHANGELOG.md

```diff
@@ -38,13 +38,16 @@ and this project aspires to adhere to [Semantic Versioning](https://semver.org/s
 - Added a `vtk` extract that saves each mesh domain to a legacy vtk file grouped, with all domain data grouped by a `.visit` file.
 - Added particle advection for streamline and related rendering support.
 - Added WarpX Streamline filter that uses charged particles.
+- Added Uniform Grid Sampling filter.
+- Added Material Interface Reconstruction (`mir`) filter which produces `cellMat` output field.
```

Comment:
Instead of cell mat, could we use the matset name as the field name?

If there are multiple matsets, one name could lead to a collision.

Reply:
Got this changed